rootfs/api/models/config.py

```diff
@@ -46,7 +46,8 @@ def _migrate_legacy_healthcheck(self):
         success_threshold = int(self.values.get('HEALTHCHECK_SUCCESS_THRESHOLD', 1))
         failure_threshold = int(self.values.get('HEALTHCHECK_FAILURE_THRESHOLD', 3))
 
-        self.healthcheck['livenessProbe'] = {
+        self.healthcheck[''] = {}
```

Comment:
I could be fine with that, and also `_all` or something that makes it descriptive but unique and not usable via Procfile or submitting it through our system in general